### DISCUSSION TITLE
No battery usage indicator in the bar in 3.2.

### DISCUSSION THREAD

**Discussion by simoncasassus** [regolith-linux/regolith-desktop#1015] [Q&A] [answered]
hi, I am testing Regolith 3.2 on my Assus Zenbook running ubuntu 24.04. The battery status does not show by default in the bar, is that normal? Apologies if this question is redundant. 
Thanks!

**Comment by kgilmer** [comment]
Hi @simoncasassus , yes it's off by default.  This is a change from the earlier in which the battery was enabled for all devices.  You can config `i3status-rs` to re-enable the battery block.

**Comment by simoncasassus** [comment]
thanks very much - this is clear.

**Comment by arnuschky** [comment, marked as answer]
For anyone else having this problem: uncomment the battery block in `/etc/regolith/i3status-rust/config.toml` to solve this.

Change is here: https://github.com/regolith-linux/i3status-rs_debian/commit/db13009a594e099ac490e179f18af620d19cbca4#diff-28043ff911f28a5cb5742f7638363546311225a63eabc365af5356c70d4deb77
